spyder_kernels/customize/spydercustomize.py

```diff
@@ -209,6 +209,16 @@ def __init__(self, *args, **kwargs):
 except:
     pass
 
+import turtle
```

Comment:
Please add a block comment above like this

```python
#==============================================================================
# Turtle adjustments
#==============================================================================
```

And then a comment below that explaining why this change is necessary and referencing the corresponding Spyder issue.